app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -465,22 +469,34 @@ export function RawUploadWizard({ studyAccession, name }) {
     }
   }
 
+  /** return a button for switching to the other experience (AnnData or Classic) */
+  function getOtherChoiceButton() {
+    const otherOption = isAnnDataExperience ? 'classic upload' : 'AnnData upload'
+    return <button
+      className="btn terra-primary-btn margin-left-30"
```

Comment:
Blocker: this button looks off in the UI.  I suspect you meant `.terra-secondary-btn`, as it seems like a better fit than `.primary-btn` and `.terra-primary-btn` is not defined.

P.S.: I hadn't appreciated this, but writing out those selectors highlights how their naming patterns are inconsistent in multiple ways!  Fixing that is out of scope here, of course; I just found it noteworthy.